packages/linea-ens-resolver/test/testL1Resolver.spec.ts

```diff
@@ -473,6 +513,125 @@ describe("Crosschain Resolver", () => {
     await rollup.setCurrentStateRoot(currentBlockNo, currentStateRoot);
   });
 
+  it("should not revert when callbackdata > 32 bytes", async () => {
+    const currentBlockNo = await rollup.currentL2BlockNumber();
+    const currentStateRoot = await rollup.stateRootHashes(currentBlockNo);
+    await rollup.setCurrentStateRoot(blockNo, stateRoot);
+    await rollup.setCurrentStateRoot(
+      blockNo + ACCEPTED_BLOCK_RANGE_LENGTH - 10,
+      stateRoot
+    );
+    let proofsEncoded = AbiCoder.defaultAbiCoder().encode(
+      [
+        "uint256",
+        "tuple(bytes key, uint256 leafIndex, tuple(bytes value, bytes[] proofRelatedNodes) proof)",
+        "tuple(bytes32 key, uint256 leafIndex, tuple(bytes32 value, bytes[] proofRelatedNodes) proof, bool initialized)[]",
+      ],
+      [blockNo, proofTest.accountProof, proofTest.storageProofs]
+    );
+    proofsEncoded = PROOF_ENCODING_PADDING + proofsEncoded.substring(2);
+
+    const result = await target.getStorageSlotsCallback(
+      proofsEncoded,
+      extraDataWithLongCallBackData
+    );
+    expect(result).to.not.be.null;
```

Comment:
Same question on what to check - I would make sure the values returned are expected even is calldata is > 32 bytes

Reply:
Ok will change